src/index.ts

```diff
@@ -15,11 +15,14 @@ import {
   annouceUserIsStreaming,
   playClip
 } from './helpers';
+import { pollCurrentGame } from './league-of-legends-api/poll-current-game';
 
 const PORT = process.env.PORT || 8081;
 const app = express();
 
-const DISCORD_BOT_TOKEN = process.env.DISCORD_BOT_TOKEN;
+const DISCORD_BOT_TOKEN: string = process.env.DISCORD_BOT_TOKEN || '';
+const IS_LOL_ANNOUNCER_ENABLED: boolean =
+  Boolean(process.env.LEAGUE_OF_LEGENDS_ANNOUNCER_ENABLED) ?? false;
```

Comment:
Thoughts on this being the first iteration for enabling polling, in a future PR refactoring to check if people in voice channels are playing league?

Reply:
Yeah I think this is solid for now. I captured a couple thoughts on what we could do long-term with this in #6 .

Reply:
Awesome thanks for doing that